share/spice/indego_bikeshare_locations/indego_bikeshare_locations.js

```diff
@@ -0,0 +1,61 @@
+(function (env) {
+    "use strict";
+
+    env.ddg_spice_indego_bikeshare_locations = function(api_result) {
+        if (!api_result || !api_result.features) {
+            return Spice.failed('indegobikesharelocations');
+        }
+
+        function normalize(item) {
+            if (item.properties.kioskPublicStatus != 'Active') {
+                return null;
+            }
+            var subtitle = item.properties.bikesAvailable + ' bike' + (item.properties.bikesAvailable  == 1 ? '' : 's') + ' | ' + item.properties.docksAvailable + ' dock' + (item.properties.docksAvailable == 1 ? '' : 's') ;
+            return {
+                id: item.properties.kioskId,
+                name: item.properties.name,
+                lat: (item.geometry.coordinates[1]).toString(),
```

Comment:
Same as in https://github.com/duckduckgo/zeroclickinfo-spice/pull/2102/files#r37313227, check if `item.geometry` exists